x-pack/plugins/security_solution/public/detection_engine/rule_management/models/coverage_overview/mitre_technique.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CoverageOverviewRuleData } from './rule_data';
+
+export interface CoverageOverviewMitreTechnique {
+  name: string;
+  /**
+   * An url leading to the technique's page
+   */
+  reference: string;
+  /**
+   * A number of covered subtechniques (having as minimum one rule enabled)
+   */
+  numOfCoveredSubtechniques: number;
```

Comment:
Do we want to treat "covered" as "having at least one installed rule associated with it AND enabled" or just "having at least one installed rule associated with it" regardless of the enabled/disabled state?

Reply:
Logically speaking `having at least one installed rule associated with it AND enabled` is the right answer.

Reply:
I agree, but we should double-check it with @approksiu